kafka-connect-redis/src/test/scala/com/datamountaineer/streamreactor/connect/redis/sink/writer/RedisSslTest.scala

```diff
@@ -90,6 +90,11 @@ class RedisSslTest extends WordSpec with Matchers with BeforeAndAfterAll with Mo
 
       }
     }
+
+//    "establish tls elasticcache" in {
```

Comment:
@andrewstevenson I suppose you forgot to remove this.

Reply:
done